libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,375 @@
+          "@id": "rockfish",
+          "Name": "Rockfish",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "triggerfish",
+          "Name": "Triggerfish",
+          "PartyId": "1"
+        }
+      ],
+      "Contest": [
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "best-animal-fish",
+          "Name": "Best Animal",
+          "NumberElected": 1,
+          "VotesAllowed": 1,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["seahorse"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["salmon"]
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "aquarium-council-fish",
+          "Name": "Zoo Council",
+          "NumberElected": 2,
+          "VotesAllowed": 2,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["manta-ray"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["pufferfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.BallotMeasureContest",
+          "@id": "fishing",
+          "Name": "Ballot Measure 3",
+          "ContestSelection": [
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "yes",
+              "Selection": "yes"
+            },
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "no",
+              "Selection": "no"
+            }
+          ],
+          "PrimaryPartyId": "1"
+        }
+      ],
+      "ElectionScopeId": ["sample-county"]
+    }
+  ],
+  "GpUnit": [
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "sample-county",
+      "Name": "Sample County",
+      "Type": "other"
+    },
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "precinct-1",
+      "Name": "Precinct 1",
+      "Type": "precinct"
+    }
+  ],
+  "CVR": [
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "UniqueId": "id-168",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
```

Comment:
Prefix our own fields with `vx`?

Reply:
cc @jonahkagan @carolinemodic

Reply:
prefixing with vx seems reasonable to me

Reply:
Yeah, let's pick a convention and make it consistent across all our CDF implementations

Reply:
OK, proposed convention:
1. Added fields are prefixed with lowercase `vx` - `vxBallotType`, `vxTestBallot`, `vxBatch`
2. Added "classes" are `CVR.vxBatch` or `BallotDefinition.vxWhatever`.